physics/src/modules/IceThermodynamicsModule/ThermoWinton.cpp

```diff
@@ -162,6 +164,11 @@ void ThermoWinton::setData(const ModelState::DataMap& state)
 
 void ThermoWinton::update(const TimestepTime& tst)
 {
+    // Advect ice temperatures
+    IIceThermodynamics::update(tst);
+    FieldAdvection::advectField(tInternal, tst, IIceThermodynamics::minT, 0.);
+    FieldAdvection::advectField(tBottom, tst, IIceThermodynamics::minT, 0.);
```

Comment:
```suggestion
    FieldAdvection::advectField(tInternal, tst, IIceThermodynamics::minT, seaIceTf);
    FieldAdvection::advectField(tBottom, tst, IIceThermodynamics::minT, seaIceTf);
```

I belatedly realised that you need to cap the internal temperatures at ``seaIceTf`` and not 0. This is because:

 - Physics: We don't want advection to cause internal melting of the ice
 - Numerics: There's a division with ``tInternal`` early on in the equations, so a zero value will generate a NaN.

Reply:
Done.